cmd/git-sync/main.go

```diff
@@ -77,6 +78,8 @@ var flSyncTimeout = pflag.Duration("sync-timeout", envDuration("GIT_SYNC_SYNC_TI
 	"the total time allowed for one complete sync, must be >= 10ms; --timeout overrides this")
 var flOneTime = pflag.Bool("one-time", envBool("GIT_SYNC_ONE_TIME", false),
 	"exit after the first sync")
+var flSleepLightly = pflag.Bool("sleep-lightly", envBool("GIT_SYNC_SLEEP_LIGHTLY", false),
```

Comment:
LOL at the name.

How about `--sync-on-signal=<signal-spec>`, where the spec is either:
* an integer for which `unix.SignalName()` returns a non-empty string
* a string for which `unix.SignalNum()` returns non-zero

We can document that as "either a valid signal name (e.g. SIGINT) or signal number (e.g. 15)".

See https://pkg.go.dev/golang.org/x/sys/unix

The default would be "" meaning, disabled.

Reply:
Also, down near the bottom of main.go is the manual-text.  Please update that with a nice entry.

Reply:
I will make an update based on SignalNum().
https://pkg.go.dev/golang.org/x/sys/unix#SignalNum